src/rendering/RenderUtil.cc

```diff
@@ -963,6 +981,31 @@ void RenderUtilPrivate::CreateRenderingEntities(
           return true;
         });
 
+    // particle emitters commands
+    _ecm.EachNew<components::ParticleEmitterCmd>(
+        [&](const Entity &_entityCmd,
+            const components::ParticleEmitterCmd *_emitterCmd) -> bool
+        {
+          // Find the particle emitter requested to be updated.
+          _ecm.Each<components::ParticleEmitter>(
+              [this, &_entityCmd, &_emitterCmd](const Entity &_entity,
+                  const components::ParticleEmitter *_emitter) -> bool
+              {
+                if (_emitterCmd->Data().name() == _emitter->Data().name())
+                {
+                  this->newParticleEmittersCmd.push_back(
+                    std::make_tuple(_entity, _emitterCmd->Data(), _entityCmd));
+                }
+                return true;
+              });
+
+          // ToDo: Remove the ParticleEmitterCmd component.
+          // Do it when https://github.com/ignitionrobotics/ign-gazebo/pull/482
+          // is merged, as it has an UpdateECM() function where we can do it.
```

Comment:
It looks like something similar is being done in https://github.com/ignitionrobotics/ign-gazebo/pull/614. We should probably make sure there are no conflicts between #482 and #614

Reply:
a07c2ba